format-specs/geoparquet.md

```diff
@@ -112,7 +112,7 @@ column (and not per geometry).
 
 This is the binary format that the geometry is encoded in.
 The string `"WKB"`, signifying Well Known Binary is the only current option, but future versions
-of the spec may support alternative encodings. This should be the ["OpenGIS® Implementation Specification for Geographic information - Simple feature access - Part 1: Common architecture"](https://portal.ogc.org/files/?artifact_id=18241) WKB representation (using codes for 3D geometry types in the \[1001,1007\] range). This encoding is also consistent with the one defined in the ["ISO/IEC 13249-3:2016 (Information technology - Database languages - SQL multimedia and application packages - Part 3: Spatial)"](https://www.iso.org/standard/60343.html) standard.
+of the spec may support alternative encodings. This SHOULD be the ["OpenGIS® Implementation Specification for Geographic information - Simple feature access - Part 1: Common architecture"](https://portal.ogc.org/files/?artifact_id=18241) WKB representation (using codes for 3D geometry types in the \[1001,1007\] range). This encoding is also consistent with the one defined in the ["ISO/IEC 13249-3:2016 (Information technology - Database languages - SQL multimedia and application packages - Part 3: Spatial)"](https://www.iso.org/standard/60343.html) standard.
```

Comment:
_Should_ this be a MUST?

Reply:
Probably.  I was wondering the same and should have asked.  Thought maybe there had been previous justification for making it a requirement instead of a recommendation.

Reply:
Yeah, to be honest I am not sure how important this is. I only know well the GEOS WKB parser, and that one is flexible in parsing both ISO and extended flavors out of the box, so the geopandas reader for geoparquet wouldn't actually care what flavor of WKB is used. 

(and we actually also need to update our writer to use the correct flavor to honor the spec!)